x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/command_and_control_telnet_port_activity.json

```diff
@@ -73,5 +73,5 @@
   "timeline_title": "Comprehensive Network Timeline",
   "timestamp_override": "event.ingested",
   "type": "query",
-  "version": 10
```

Comment:
This is ok - from the manually reconciled version bumps elastic/detection-rules/pull/1950